tests/components/mqtt/test_alarm_control_panel.py

```diff
@@ -65,7 +65,7 @@
         "name": "test",
         "state_topic": "alarm/state",
         "command_topic": "alarm/command",
-        "code": "1234",
+        "code": "0123",
```

Comment:
Does this mean that 1234 as a code no longer works ?

Reply:
Yes, it will. I changed it to 0123, to ensure its working. (0123 = 123 as a number)